src/AppBundle/Resources/views/Site/edit.html.twig

```diff
@@ -10,6 +10,176 @@
     </div>
 {% endblock headerPanel %}
 
-{% block body -%}
+{% block body %}
     {{ form(edit_form) }}
 {% endblock %}
+
+{% block javascripts %}
+    <script>
```

Comment:
it is used only while creating/editing site, but gulp concat it in 1 js file in final